### DISCUSSION TITLE
Auto issue names in task lists don't appear in Project Board preview

### DISCUSSION THREAD

**Discussion by mungojam** [community/community#5281] [Projects and Issues] [answered]
The new automatic displaying of issue titles when including the issue number in lists is great. It works well in issues and PRs. But when you click on an issue on a project board, the preview on the right does not show the title, so you are just left with the issue number. This is a pain when reviewing the project board.

**Comment by mungojam** [comment]
Oddly, the preview shows if I tick any of the boxes in the list within the preview. So it seems this is half working

**Comment by dillingham** [comment]
Agreed, not ideal at all. Here is a screenshot for onlookers 
<img width="760" alt="Screen Shot 2021-09-14 at 12 39 13 PM" src="https://user-images.githubusercontent.com/29180903/133298775-eeb5d1e0-670e-46b8-b436-c892c340c472.png">

**Comment by mungojam** [comment, marked as answer]
Pretty sure this is sorted now
